tools/please_diff_graphs/plz_diff_graphs_main.go

```diff
@@ -69,6 +73,7 @@ func readStdin() []string {
 func main() {
 	cli.ParseFlagsOrDie("Please graph differ", "9.1.2", &opts)
```

Comment:
Is this a version number?

Reply:
Yeah. The various tools all got them at one point, but I later realised that was a bad idea (because upping the version number causes them all to change at each release which causes unnecessary rebuilds).

One day I should just get rid of them all again...